### PR TITLE
Ivory 358 eligibility checker flow

### PR DESCRIPTION
**Pull request by Jozzey**
Created the eligibility checker flow. The pages have no content yet or unit tests. This will be added when the content is available.

Moved the code that calculates & stores the "cost" from "what-type-of-item-is-it" to "can-continue" so that it is also calculated when the user goes via the eligibility checker route.

Renamed/removed the generic yes-no-idk that was used by 2 routes and replaced with 2 routes that follow the standard naming convention we are using. I don't think that generic page was really adding any value. Using our standard naming convention makes it easier to know which views/routes go together and the effort saved by using that generic page was minimal.

### CODE REVIEW

**Review by duncannixon — CHANGES_REQUESTED** [review]
Everywhere where you have code like this:
      case 'Yes':
        return ...
      case 'No':
        return ...
      case 'I dont know':
        return ...
        
     It would be better to use the Yes / No in Constants (and add an 'I don't know' constant):

const Options = {
  YES: 'Yes',
  NO: 'No'
}

File: server/routes/eligibility-checker/cannot-continue.route.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+const handlers = {
+  get: (request, h) => {
+    return h.view(Views.CANNOT_CONTINUE, {
+      ..._getContext()
+    })
+  },
+
+  post: (request, h) => {
+    return h.redirect('https://www.gov.uk/')
```
Comment (duncannixon) [inline server/routes/eligibility-checker/cannot-continue.route.js:13]: All of the occurrences of this URL should be defined in the Constants file

**Comment by Jozzey** [comment]
Application updated as per review comments

**Review by duncannixon — APPROVED** [review]
Good work!

**Pull request merged**